apps/weathermap/weather_map.star

```diff
@@ -351,7 +342,7 @@ def main(config):
     Returns:
         A definition of what to render.
     """
-    response = http.get(url = WEATHER_MAPS_URL)
+    response = http.get(url = WEATHER_MAPS_URL, ttl_seconds = 30)
```

Comment:
FYI - I added a slight cache here. Our new caching client caches for a minimum of 5 seconds, which has smoothed over this request volume. We were hitting around 20 rps for this app before hand 🙀